_data/beyond-sidebar.yml

```diff
@@ -804,9 +804,17 @@
             - title: Remove multiple products from product categories
               link: https://api-docs.beyondshop.cloud/ng-product-management-product-management-public-api.html#resources-categories-batch-remove-products
               id: remove_multiple_products_from_product_categories
+        - title: Product category-related searches
+          entries:
             - title: Search product categories
               link: https://api-docs.beyondshop.cloud/ng-product-management-product-management-public-api.html#resources-categories-search
               id: search_product_categories
+            - title: Find used category tags
+              link: https://api-docs.beyondshop.cloud/ng-product-management-product-management-public-api.html#resources-find-category-tags
+              id: find_used_category_tags
+            - title: Find category manufacturers
+              link: https://api-docs.beyondshop.cloud/ng-product-management-product-management-public-api.html#resources-find-category-manufacturers
+              id: find_category_manufacturers              
```

Comment:
@dkluever22 According to CircleCi this line has a lot of whitespaces. Please remove all from  the last character

Reply:
@gersanco Thank, German! I've corrected it and now everything looks fine :)